mist-core/src/main/java/edu/snu/mist/core/master/MistMaster.java

```diff
@@ -118,6 +134,13 @@ private MistMaster(
       taskRequestor.recoverTaskConn();
       masterSetupFinished.setFinished();
     }
+    if (dynamicScalingEnabled) {
+      // Start dynamic scaling monitoring.
+      final ScheduledExecutorService dynamicScalingService = Executors.newSingleThreadScheduledExecutor();
+      final Injector injector = tang.newInjector();
+      final DynamicScalingPolicy policy = injector.getInstance(DefaultDynamicScalingPolicy.class);
```

Comment:
How about creating `DynamicScalingManager` and injecting it? And do we only consider the periodic policy? can't we implement on-demand scale in/out?